prefetch.bs

```diff
@@ -249,11 +302,21 @@ These algorithms are based on [=process a navigate fetch=].
 
             <div class="issue">It might be possible to "downgrade" to [=uncredentialed prefetch=] in this case.</div>
         1. Otherwise, return "`Allowed`".
-    1. Let (|response|, |locationURL|) be the result of [=performing a common navigational fetch=] given |request|, "`other`", |browsingContext|, null, an empty algorithm, |shouldBlockNavigationRequest|, and an empty algorithm.
+
+    1. Let |shouldBlockNavigationResponse| be the following steps, given [=request=] |request| and [=response=] |response|:
+        1. Let |responseCOOP| be the result of [=obtaining a cross-origin opener policy=] given |response| and |request|'s [=request/reserved client=].
+        1. If |sandboxFlags| is not empty and |responseCOOP|'s [=cross-origin opener policy/value=] is not "`unsafe-none`", then return "`Blocked`".
+        1. [=list/Append=] |response| to |redirectChain|.
+
+            <div class="note">This allows [=enforcing a response's cross-origin opener policy=] to be deferred.</div>
```

Comment:
Why do we want to defer?

Reply:
Expanded to explain that enforcing has side effects (queuing violation reports) that are not desirable at prefetch time.